README.md

```diff
@@ -17,22 +17,22 @@ or to update
 
 	go get -u gopkg.in/bluesuncorp/validator.v5
 
-And then just import the package into your own code.
+Then import the validator package into your own code.
 
 	import "gopkg.in/bluesuncorp/validator.v5"
 
-Usage
-=====
+Usage and documentation
+=======================
 
 Please see http://godoc.org/gopkg.in/bluesuncorp/validator.v5 for detailed usage docs.
 
-Contributing
-============
+How to Contribute
+=================
 
-There will be a development branch for each version of this package i.e. v1-development, please
-make your pull requests against those branches.
+There will be always be a development branch for each version i.e. `v1-development`. In order to contribute, 
+please make your pull requests against those branches.
 
-If changes are breaking please create an issue, for discussion and create a pull request against
+If changes made fails the test or are broken, please create an issue, for discussion and create a pull request against
```

Comment:
I will update this verbiage as the change isn't relaying the information intended